operator/k8s_identity.go

```diff
@@ -52,6 +53,20 @@ func deleteIdentity(ctx context.Context, identity *v2.CiliumIdentity) error {
 	return err
 }
 
+func updateIdentity(ctx context.Context, identity *v2.CiliumIdentity) error {
+	_, err := ciliumK8sClient.CiliumV2().CiliumIdentities().Update(
+		ctx,
+		identity,
+		metav1.UpdateOptions{})
+	if err != nil {
+		log.WithError(err).Error("Updating Identity")
+	} else {
+		log.WithField(logfields.Identity, identity.GetName()).Info("Updated identity")
```

Comment:
Does this need to be `loglevel=info`? It might be a bit noisy in large clusters.

Reply:
Good point.